po/zh_CN.po

```diff
@@ -313,6 +313,10 @@ msgstr "长日期格式：YYYY年MM月DD日"
 msgid "Show shortdate"
 msgstr "短日期格式：YYYY-MM-DD"
 
+#: ../applets/clock/clock.ui.h:37
```

Comment:
We have a file conflict with master branch at this point.
Better do not edit this file. Every pull from our translation sever (transifex) will override this file.
We will update mate-panel.pot file with your changes after this is merged.
After that the translators can update this translation string.